tests/private_contribution_bounds_test.py

```diff
@@ -202,9 +202,6 @@ def test_score_gaussian_noise_invalid_values_throws_exception(self):
         with self.assertRaises(RuntimeError):
             l0_scoring_function.score(-1)
 
```

Comment:
There is still warning

"  /home/runner/work/PipelineDP/PipelineDP/pipeline_dp/dp_computations.py:103: RuntimeWarning: invalid value encountered in sqrt
    return np.sqrt(l0_sensitivity) * linf_sensitivity"

we should remove l0_scoring_function.score(-1)